tests/test_utils.py

```
@@ -161,3 +161,37 @@ def print_performance_summary(
+        # Batch результаты
+        assert isinstance(results, list)
+        assert len(results) == expected_count
+
+        for result in results:
+            _validate_single_result(result, check_timestamps)
+    else:
+        # Одиночный результат
+        _validate_single_result(results, check_timestamps)
+
+
+def _validate_single_result(result, check_timestamps=True):
```

Ruff (0.11.9)

Missing return type annotation for private function `_validate_single_result`

Add return type annotation: `None`

(ANN202)